modules/server_new/src/main/scala/observe/server/gmos/Gmos.scala

```diff
@@ -187,7 +187,7 @@ object Gmos {
   def shutterStateObserveType(obsType: StepType): ShutterState = obsType match {
     case StepType.DarkOrBias(_) | StepType.ExclusiveDarkOrBias(_) | StepType.DarkOrBiasNS(_) =>
       ShutterState.CloseShutter
-    case _                                                                                   => ShutterState.UnsetShutter
+    case _                                                                                   => ShutterState.OpenShutter
```

Comment:
What does it mean `UnsetShutter`? Is it ever used?

Reply:
I don't think so. In the old model the value was a string, and the line `case _ => ShutterState.UnsetShutter` was there just to complete the pattern